src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4185,7 +4185,10 @@ .FORWARDHELPCATEGORY Cmdlet
             $customPagerCommandLine = $env:PAGER
 
             # Split the command line into tokens, respecting quoting.
-            $customPagerCommand, $customPagerCommandArgs = & { Write-Output -- $customPagerCommandLine }
+            $errs = $null
+            $tokens = [System.Management.Automation.PSParser]::Tokenize($customPagerCommandLine, [ref]$errs)
+            $customPagerCommand = $tokens[0].Content
+            $customPagerCommandArgs = $tokens[1].Content
```

Comment:
What if there is some arguments?

Also now we should change
`$cmds = Get-Command $customPagerCommand, $customPagerCommandLine -ErrorAction Ignore`
with 
`$cmds = Get-Command $customPagerCommand-ErrorAction Ignore`